security/try-node-to-node-encryption.md

```diff
@@ -1,70 +1,68 @@
 ---
 title: Try out the node-to-node encryption tech preview
-description: Try out enabling WireGuard for state-of-the-art cryptographic security between pods for Calico clusters.
+description: Try out enabling WireGuard for state-of-the-art cryptographic security between nodes for Calico clusters.
 ---
 
 ### Big picture
 
 Enable WireGuard to secure node-to-node traffic in a {{site.prodname}} cluster.
 
-> **Warning!** Node-to-node encryption is a tech preview and should not be used in production clusters. It has had very limited testing and it will contain bugs (please report these on the Calico Users Slack or GitHub). This feature is currently not supported with overlay networks (IP in IP, VXLAN) due to known issues with NodePort services.
-{: .alert .alert-danger }
+> **Warning!** Node-to-node encryption is a tech preview and should not be used in production clusters. It has had very limited testing and it will contain bugs (please report these on the {{site.prodname}} Users Slack or GitHub). This feature is currently not supported with overlay networks (IP in IP, VXLAN) due to known issues with NodePort services.
```

Comment:
The change to {{site.prodname}} isn't good in this case. We don't have "Calico Enterprise Users Slack", we only have "Calico Users Slack".